aws/resource_aws_ebs_snapshot_copy.go

```diff
@@ -116,7 +116,7 @@ func resourceAwsEbsSnapshotCopyRead(d *schema.ResourceData, meta interface{}) er
 		SnapshotIds: []*string{aws.String(d.Id())},
 	}
 	res, err := conn.DescribeSnapshots(req)
-	if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "InvalidSnapshotID.NotFound" {
+	if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "InvalidSnapshot.NotFound" {
```

Comment:
These can be simplified using `isAWSErr(err, CODE, MESSAGE)` in the future, e.g.

```suggestion
	if isAWSErr(err, "InvalidSnapshot.NotFound", "") {
```

Reply:
fixed.